examples/experimental/export_example.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Nested context managers.
</pre><p>what is the point of showing this nested context manager? what functionality is displayed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1346/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It was for testing but I see that if this is a generic export example, a more interesting/more relevant recording is warranted. Will work on that. </p><p> </p>